Thorium-Sim/thorium-nova
# Navigation Card: Add waypoint

## Description

Makes it possible to add waypoints from the Navigation card.

## Related Issue








Closes #387

## How do you know the changes work correctly?






Manual Test:
- Start a flight
- Go to the navigation card
- Click an object
- Click "Add Waypoint"
- The waypoint should appear.

## Types of changes



      change)

## Checklist:
